File: keepr-server/Models/Vault.cs
namespace keepr.Models
{
    public class Vault
    {
         public Vault()
      {
      }

      public int Id { get; set; }
      public string Name { get; set; }
      public string Description {get;set;}
      public bool isPrivate {get;set;}
      public string creatorId {get;set;}
      public Profile Creator{get;set;}
    }
}

File: keepr-client/src/models/Vault.js
export class Vault {
  constructor(data = {}) {
    this.id = data.id || data._id || 'placeholder id'
    this.creatorId = data.creatorId || 'test creatorId'
    this.name = data.name || 'b'
    this.description = data.description
    this.isPrivate = data.isPrivate
  }
}
